qa/L0_sequence_batcher/test.sh

```diff
@@ -827,8 +838,14 @@ fi
 
 ### Start Preserve Ordering Tests ###
 
-# Test only supported on windows currently due to use of python backend models
-if [ ${WINDOWS} -ne 1 ]; then
+# FIXME: Test only supported on windows currently due to use of python backend models.
+# Now that Windows supports the PYBE, we should check that this tests works once Windows
+# CI is stable.
+
+# These subtests use python models. They should not be executed if 'python' is not one
+# of the backends under test.
+should_test_backend "python"
+if [ $? -eq 1 ] && [ ${WINDOWS} -ne 1 ]; then
```

Comment:
Using the return value on a bash function (i.e. `$?`) is a bit dubious imo. I am more of the opinion of setting variables and then inspecting those. Perhaps @mc-nv has an opinion here?

Reply:
I completely agree, but it seems that in bash return values can only be accessed from `$?` (source: https://linuxize.com/post/bash-functions/#return-values).
Since writing this, I realize I can probably just do something like `echo "true"` and store it like `run_test=$(should_test_backend "python")`

Reply:
Recently I used following to determine either it windows or not, may be it can be suitable here...
```suggestion
if [[ -v WSL_DISTRO_NAME ]] || [[ -v MSYSTEM ]];; then
```

Reply:
We want the exact opposite check, but point taken and the code has been updated.